libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -825,6 +825,13 @@ static DNNReturnType execute_model_tf(const DNNModel *model, const char *input_n
             out_frame->height = output[0].height;
```

Comment:
there are typo (wrong words) in commit log.

Reply:
Sorry, i just found 'examplei', what is the other one?

Reply:
There are two dogs detected with 'core' 94.05% and 93.45%.